src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -186,6 +186,70 @@ def _check_state_after_step_update(
             )
         return self
 
+    def get_jobs(
+        self,
+    ) -> Dict[Tuple[str, str, str], "Job"]:
+        if self._snapshot:
+            return self._snapshot.get_jobs()
+        return {}
+
+    def get_job_states_for_all_reals_and_steps(
+        self,
+    ) -> Dict[Tuple[str, str, str], Union[str, datetime.datetime]]:
+        if self._snapshot:
+            return self._snapshot.get_job_states_for_all_reals_and_steps()
+        return {}
+
+    @property
+    def reals(self) -> Dict[str, "RealizationSnapshot"]:
+        return {
+            real_id: RealizationSnapshot(**real_data)
+            for real_id, real_data in self._realization_states.items()
+        }
+
+    def get_real_ids(self) -> Sequence[str]:
+        """we can have information about realizations in both _realization_states and
+        _job_states - we combine the existing IDs"""
+        real_ids = []
+        for idx in self._job_states:
+            real_id = idx[0]
+            if real_id not in real_ids:
+                real_ids.append(real_id)
+        for real_id in self._realization_states:
+            if real_id not in real_ids:
+                real_ids.append(real_id)
+        return sorted(real_ids, key=int)
+
+    @property
+    def metadata(self) -> Dict[str, Any]:
+        return self._metadata
+
+    def get_jobs_for_real_and_step(
```

Comment:
Yes, there seems to more methods that are unused everywhere except for in the unit tests. According to vulture, it could be 363 variables/functions/methods in total. Should I remove the unused methods in the updated classes in this PR, and then create a new issue for the other ones?

Reply:
I would save all that cleanup for a new PR :)